MetaMask/metamask-extension
# Show a slippage notification again before submitting a swap

## Explanation
We already show users a warning notification if they choose a slippage that is too low or too high. Now we will also show a slippage modal, which a user has to confirm before submitting a swap.

I've also improved content for our slippage notifications to make them easier to understand.

## Screenshots/Screencaps
Low slippage notification in Transaction Settings
![image](https://github.com/MetaMask/metamask-extension/assets/80175477/55461c1d-cbbd-4de5-8ad1-85d32b77de11)

High slippage notification in Transaction Settings
![image](https://github.com/MetaMask/metamask-extension/assets/80175477/b29c8413-f4be-47dd-a735-f31e2b927642)

Low slippage modal
![image](https://github.com/MetaMask/metamask-extension/assets/80175477/188470ed-309f-49e8-a2c0-a1248d4d4477)

High slippage modal
![image](https://github.com/MetaMask/metamask-extension/assets/80175477/7558cb72-150d-4963-a780-ed29a349eb22)

## Manual Testing Steps
Low slippage modal
- Open Swaps
- Click on Transaction Settings and set custom slippage to e.g. 0.5
- Get a quote
- Click on the Swap button and you will see the low slippage modal

High slippage modal
- Open Swaps
- Click on Transaction Settings and set custom slippage to e.g. 10
- Get a quote
- Click on the Swap button and you will see the high slippage modal
